src/targeting/Semgrepignore.ml

```diff
@@ -59,6 +59,9 @@ type exclusion_mechanism = Gitignore_and_semgrepignore | Only_semgrepignore
 *)
 let builtin_semgrepignore_for_semgrep_scan =
   {|
+# Git administrative folder or file
```

Comment:
could be good to add why we put this folder given git --others is skipping this directory already
and why pysemgrep does not need this .git in his own .semgrep_ignore template file.